packages/client/tests/e2e/typed-sql-query-compiler-adapter-libsql/prisma/schema.prisma

```diff
@@ -1,7 +1,9 @@
 generator client {
-  provider        = "prisma-client"
-  previewFeatures = ["typedSql", "queryCompiler", "driverAdapters"]
-  output          = "../src/generated/prisma"
+  provider            = "prisma-client"
+  previewFeatures     = ["typedSql", "queryCompiler", "driverAdapters"]
+  output              = "../src/generated/prisma"
+  moduleFormat        = "commonjs"
+  importFileExtension = ""
```

Comment:
This change is necessary because our heuristics for detecting the default aren't checking the `type` property in `package.json` and always choose ESM by default when `moduleResolution = "nodenext"`.